### DISCUSSION TITLE
Custom color to BottomSheetBackdrop

### DISCUSSION THREAD

**Discussion by hanselabreu** [gorhom/react-native-bottom-sheet#431] [Q&A] [answered]
Hey guys,

Is there a way to pass a custom color to the `BottomSheetBackdrop`. Doing the following does not appear to be modifying the background color:

```jsx
<BottomSheetBackdrop style={{ backgroundColor: "red" }} />
```

Is the color defined in a different property or what am I doing wrong?

P.S. I need this in order to improve contrast when changing between `light/dark` theme.

**Comment by hanselabreu** [comment, marked as answer]
This is a bug should be fixed once #448 gets merged.

**Reply by gorhom** [reply]
thanks @hanselabreu for your conitrbution ! your fix is already out with `v3.6.5` https://github.com/gorhom/react-native-bottom-sheet/releases/tag/v3.6.5

**Reply by hanselabreu** [reply]
@gorhom thanks for the prompt release! 🙂

**Comment by jocoders** [comment]
I have @gorhom/bottom-sheet": "^4" version, but still is not possible to change backdrop color by this way: 
```
<BottomSheetBackdrop style={{ backgroundColor: "red" }} />
```
Any ideas how can I change backdrop color?

**Reply by personable** [reply]
@jocoders It seems like adding that `style` prop erases the existing backdrop styles. I got it working like this:
```
        style={{
          backgroundColor: tokens.color_background_backdrop,
          position: "absolute",
          top: 0,
          bottom: 0,
          right: 0,
          left: 0,
        }}
```

**Reply by SMhdAsadi** [reply]
You should override the existing styles:
```tsx
<BottomSheetBackdrop
  {...props}
  style={[
    props.style,
    {backgroundColor: 'red'},
  ]}
/>
```

**Reply by personable** [reply]
Ah thanks @SMhdAsadi !
